### PR TITLE
Upgrade Django compatibility to 1.9 - 1.11.

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko

File: tests/urls.py [hunk 1]
```diff
@@ -2,6 +2,6 @@
 from django.contrib import admin
 
 urlpatterns = [
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^admin/', admin.site.urls),
```
Comment (ldko) [inline tests/urls.py]: Should we remove the admin pattern? If we keep it, I think we need to add a `DATABASES` setting, so the admin url won't error if someone is running this site for development purposes.

**Review by ldko — COMMENTED** [review]
We need to modify Django version requirements in README.md.

File: tests/urls.py (hunk 1 above)
Comment (somexpert) [reply tests/urls.py]: Think we should just get rid of it. This app has no models, no sign-in, or anything that would warrant an admin site.

**Review by ldko — APPROVED** [review]

**Pull request merged**